Contents/Code/default_prefs.py

```diff
@@ -20,6 +20,7 @@
     enum_webapp_locale='en',
     str_webapp_http_host='0.0.0.0',
     int_webapp_http_port='9494',
+    bool_webapp_require_login='False',
```

Comment:
I put an option for that in the idea that some people don't use the builtin auth mechanisms for stuff like internal services and instead rely on a gateway/reverse proxy handling all of that (meaning internal services all assume the admin account is "always logged in"). We could enable it by default, but I think it doesn't cost much to allow the user to disable it

Reply:
Sorry for the delay in my response on this. So the main point of adding authentication is to be able to add features to the UI that require authentication. For example, a user may need to remove a theme that was uploaded by Themerr so that Plex will then pick up the `theme.mp3` they've dropped into the items directory.

And I understand your point about handling authentication via other means, but if it's not somehow controlled through Themerr then there is no way to ensure the user is authenticated... and 99% of users are not conscious of security issues and will disable the authentication for convenience without understanding the risks.

Reply:
The security risk is real, I agree with you on that. Would you be okay to keep the option but hide it from the Plex plugin UI, while still allowing it to be disabled by manually editing the config file? I think it's reasonable to assume someone tech-savvy enough to edit a config file deep down in the Plex folder knows what the security implications are of not requiring auth.

Reply:
A hidden setting is okay with me.